documentation/src/main/asciidoc/topics/config_examples/server_network_default.xml

```diff
@@ -1,3 +1,5 @@
+<!-- Maps a single public interface to the IPv4 loopback IP address, 127.0.0.1.
```

Comment:
actually, you can map to any IP address
what about `Selects a specific IPv4 address (can be public, private, or loopback)`